source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -146,7 +146,7 @@ connection behavior:
    * - ``writeConcern()``
      - | Sets the :manual:`write concern </reference/write-concern/>`.
        |
-       | **Default**: ``"majority"``
+       | **Default**: ``WriteConcern#ACKNOWLEDGED``
```

Comment:
Consider linking to https://www.mongodb.com/docs/manual/reference/write-concern/#implicit-default-write-concern

Reply:
Will do.